tests/integration/defs/cpp_common.py

```diff
@@ -477,6 +487,90 @@ def run_disagg_multi_gpu_tests(build_dir: _pl.Path):
         leader_commands=[f"--gtest_output=xml:{xml_output_file}"])
     run_command(trt_model_test, cwd=tests_dir, env=mgpu_env, timeout=1500)
 
+    # UCX transceiver tests, the test may not be built if ENABLE_UCX is 0
+    new_env = copy.copy(cpp_env)
+    new_env["TRTLLM_USE_UCX_KVCACHE"] = "1"
+    xml_output_file = build_dir / "results-multi-gpu-disagg-executor-2-process.xml"
+    trt_model_test = produce_mpirun_command(
```

Comment:
Looks like all the tests below rely on MPI to launch the disagg executor test. If the goal of this MR is remove the dependency of launching the executor instances with MPI, should we add integration tests that don't rely on MPI for launching the executor instances?